puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/nightlyTest.groovy

```diff
@@ -0,0 +1,42 @@
+pipeline {
+    agent { label 'foreman' }
+
+    stages {
+        stage('Setup Environment') {
+            steps {
+                deleteDir()
+                git url: 'https://github.com/theforeman/forklift.git'
+            }
+        }
+        stage('Provision Node') {
+            steps {
+                provision()
+            }
+        }
+        stage('Install Pipeline Requirements') {
+            steps {
+                runPlaybook('playbooks/setup_forklift.yml', cico_inventory('./'), [], ['-b'])
+            }
+        }
+        stage('Run Pipeline') {
+            steps {
+                duffy_ssh("cd forklift && ansible-playbook pipelines/pipeline_katello_nightly.yml -e forklift_state=up", 'duffy_box', './')
```

Comment:
I don't see a `forklift_state=destroy` to clean it up. Does `deprovision()` take care of this?

Reply:
Yes, deprovision wipes the duffy box so no need to destroy it earlier. Also it lets my collect_debug script later collect bats_results and sosreport.